Sources/Turf/Consumer/Consumer+WKT.swift

```diff
@@ -0,0 +1,327 @@
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return coords
+        case .coordinatesArray2D:
+            guard let coords = values as? [[LocationCoordinate2D]] else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return coords
+        case .coordinatesArray3D:
+            guard let coords = values as? [[[LocationCoordinate2D]]] else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return coords
+        case .point:
+            guard values.count > 1 else { return nil }
+            guard let coords = values[1] as? LocationCoordinate2D else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return Point(coords)
+        case .multiPoint:
+            guard values.count > 1 else { return nil }
+            let coords = values[1..<values.endIndex].compactMap { $0 as? LocationCoordinate2D }
+            guard coords.count == values.count - 1 else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return MultiPoint(coords)
+        case .lineString:
+            guard values.count > 1 else { return nil }
+            guard let coords = values[1] as? [LocationCoordinate2D] else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return LineString(coords)
+        case .multiLineString:
+            guard values.count > 1 else { return nil }
+            guard let coords = values[1] as? [[LocationCoordinate2D]] else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return MultiLineString(coords)
+        case .polygon:
+            guard values.count > 1 else { return nil }
+            guard let coords = values[1] as? [[LocationCoordinate2D]] else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return Polygon(coords)
+        case .multiPolygon:
+            guard values.count > 1 else { return nil }
+            guard let coords = values[1] as? [[[LocationCoordinate2D]]] else {
+                throw WKTError.coordinatesParsingFailed(values)
+            }
+            return MultiPolygon(coords)
+        case .geometryCollection:
+            guard values.count > 1 else { return nil }
+            let geometries = values[1..<values.endIndex].compactMap { ($0 as? GeometryConvertible)?.geometry }
+            guard geometries.count == values.count - 1 else {
+                throw WKTError.geometriesParsingFailed(values)
+            }
+            return GeometryCollection(geometries: geometries)
+        case .object:
+            return values.first
+        }
+    }
+}
+
+
+extension Consumer where Label == WKTParser.WKTLabel {
+    private static var space: Consumer<WKTParser.WKTLabel> {
+        return .discard(.zeroOrMore(.character(in: " \t\n\r")))
```

Comment:
Are these the only whitespace characters to consider, or should we consider all of `CharacterSet.whitespacesAndNewlines`?